src/Compilers/CSharp/Portable/Declarations/DeclarationModifiers.cs

```diff
@@ -36,8 +36,12 @@ internal enum DeclarationModifiers : uint
         Async = 1 << 20,
         Ref = 1 << 21, // used only for structs
 
-        All = (1 << 23) - 1, // all modifiers
-        Unset = 1 << 23, // used when a modifiers value hasn't yet been computed
+
+        // PROTOTYPE(ft): leaving 22 free since required is using it
+        File = 1 << 23, // used only for structs
```

Comment:
Used only for 'structs'?

Reply:
This was a copy-paste error. It should probably read as:

```suggestion
        File = 1 << 23, // used only for types
```